### PR TITLE
Add DevSecOps

### PR DESCRIPTION
**Pull request by JakobRPennington**
https://github.com/TaptuIT/awesome-devsecops

This list is a collection of resources and tooling related to DevSecOps. DevSecOps is an extension of DevOps - a methodology and philosophy for software development - that aims to incorporate security expertise and testing into the development lifecycle, with the goal of creating more secure software.

This list of resources can help software development teams learn about and adopt DevSecOps practices and tooling, resulting in more secure software for everybody.

Reviewed PRs: #1756 #1758 

By submitting this pull request I confirm I've read and complied with the below requirements 🖖

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by JakobRPennington** [comment]
unicorn

**Comment by mcauser** [comment]
awesome-lint
  ✖  1:1  Missing newline character at end of file  remark-lint:final-newline

**Comment by JakobRPennington** [comment]
I have double-checked the readme to make sure there is a newline at the end re-ran the linter which returned no issues.

Based on your suggestion in the [MicroPython PR](https://github.com/sindresorhus/awesome/pull/1776#issuecomment-629749401), I removed the 'Awesome DevSecOps heading as it is repeated in the banner image.

**Comment by JakobRPennington** [comment]
Hi @sindresorhus,

A appreciate working through all these PRs must be super time consuming, so I hope the changes above have resolved the conflicts with the list guidelines.

Please let me know if there's anything else I can do 😃

**Comment by sindresorhus** [comment]
> The image should be high-DPI. Set it to maximum half the width of the original image.

---

> Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

---

`DevSecOps` in the description should be linkified to a canonical about `DevSecOps`.

File: readme.md [hunk 1]
```diff
@@ -501,6 +501,7 @@
 - [Fuzzing](https://github.com/cpuu/awesome-fuzzing#readme) - Automated software testing technique that involves feeding pseudo-randomly generated input data.
 - [Embedded and IoT Security](https://github.com/fkie-cad/awesome-embedded-and-iot-security#readme)
 - [GDPR](https://github.com/bakke92/awesome-gdpr#readme) - Regulation on data protection and privacy for all individuals within EU.
+- [DevSecOps](https://github.com/TaptuIT/awesome-devsecops) - Integration of security practices into DevOps.
```
Comment (sindresorhus) [inline readme.md:504]:
```suggestion
- [DevSecOps](https://github.com/TaptuIT/awesome-devsecops#readme) - Integration of security practices into [DevOps](https://en.wikipedia.org/wiki/DevOps).
```

**Comment by sindresorhus** [comment]
I decided to rename `master` branch to `main`, and apparently that permanently closes all PRs. Ugh. Sorry about that. Can you reopen this PR targeting the `main` branch? Include a link to this PR in the new one, so we can quickly see what was discussed previously.

**Pull request closed without merging**